contrib/base64-cmake/config-header.tpl

```diff
@@ -0,0 +1,5 @@
+#define HAVE_AVX                   0
```

Comment:
As I see, the library does CPU features detection at runtime.
https://github.com/aklomp/base64/blob/master/lib/codec_choose.c#L179

For example, it should use AVX on CPUs with AVX support even if ClickHouse was compiled without AVX.
Is that true?

Reply:
Implemented compile-time detection. Runtime is up to libray.